lib/iris/experimental/um.py

```diff
@@ -687,6 +687,18 @@ def normalise(values, kind):
             sector_size = self._WORDS_PER_SECTOR * self._word_size
             for field in self.fields:
                 data = field.get_data()
+
+                if hasattr(field, "lbpack"):
+                    lbpack = field.lbpack
+                    packing = lbpack % 10
+                    compression = (lbpack // 10) % 10
+                    number_format = (lbpack // 1000) % 10
+                    if (packing not in (0, 1) or
```

Comment:
For writing of data, that isn't (yet) the case - we cannot currently write packed data. Because of the overlap with this and the work that @pp-mo is doing, the tests are by far the more interesting piece of work.